api/api.proto

```diff
@@ -1543,6 +1543,23 @@ message SdkSchedulePolicyIntervalPeriodic{
 
 // Defines a schedule policy interval
 message SdkSchedulePolicyInterval {
+
+  enum DefaultDailyScheduleRetain {
```

Comment:
We need documentation in all of these. For the ones with UNKNOWN you do not need to prefix them. just call them `UNKOWN` and add a comment `Do not use`.

Reply:
can only add UNKOWN once , gives error when i add again.
Hence i had to prefix